### PR TITLE
APPS-2387 - Add Citation Count

### PR DESCRIPTION
**Pull request by bergieboy**
Added Citation Count to the 'Health And Safety'  portion of 'Provider Details'. Also added info tooltip with a description of the field. 
Screen Shot 2020-10-30 at 9 52 00 AM
Screen Shot 2020-10-30 at 9 52 29 AM

### CODE REVIEW

**Review by anzioka — APPROVED** [review]

File: src/containers/location/providers/ProviderDetailsContainer.js [hunk 1]
```diff
@@ -5,10 +5,13 @@ import React, { Fragment } from 'react';
 import moment from 'moment';
 import styled, { css } from 'styled-components';
 import { Map, List } from 'immutable';
-import { Colors } from 'lattice-ui-kit';
+import { Colors, Tooltip } from 'lattice-ui-kit';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
 
+import { faInfoCircle } from '@fortawesome/pro-light-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```
Comment (UnsungHero97) [inline src/containers/location/providers/ProviderDetailsContainer.js:13]: Please run the import sorter.

File: src/containers/location/providers/ProviderDetailsContainer.js [hunk 2]
```diff
@@ -5,10 +5,13 @@ import React, { Fragment } from 'react';
 import moment from 'moment';
```
Comment (UnsungHero97) [inline src/containers/location/providers/ProviderDetailsContainer.js:5]: No change required. `moment` needs to be removed.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/containers/location/providers/ProviderDetailsContainer.js [hunk 3]
```diff
@@ -412,7 +424,19 @@ class ProviderDetailsContainer extends React.Component<Props> {
               {lastInspectionDate}
             </DataRows>
           </Row>
-          {complaints}
+          <Row>
+            <FlexContainer>
+              {renderText(LABELS.CITATIONS)}
+              <MarginWrapper>
+                <Tooltip arrow placement="top" title={renderText(LABELS.CITATIONS_INFO)}>
+                  <div>{ infoIcon }</div>
```
[the remark this answers is not in the document]
Comment (bergieboy) [reply src/containers/location/providers/ProviderDetailsContainer.js:432]: https://material-ui.com/components/tooltips/#custom-child-element
The tooltip needs to apply DOM event listeners to its child element. If the child is a custom React element, you need to make sure that it spreads its properties to the underlying DOM element.

I'll modify to:
```
const InfoIcon = React.forwardRef((props, ref) => (
  <span {...props} ref={ref}>
    <FontAwesomeIcon icon={faInfoCircle} size="sm" />
  </span>
));
```
...
```
<Tooltip arrow placement="top" title={renderText(LABELS.CITATIONS_INFO)}>
    <InfoIcon />
 </Tooltip>
```

[the remark this answers is not in the document]
Comment (UnsungHero97) [reply src/containers/location/providers/ProviderDetailsContainer.js:432]: Are you sure this is necessary? Doesn't this just work?

```
<Tooltip arrow placement="top" title={...}>
  <FontAwesomeIcon icon={faInfoCircle} size="sm" />
</Tooltip>
```

**Review by UnsungHero97 — APPROVED** [review]

**Review by anzioka — APPROVED** [review]

**Pull request merged**